### ISSUE TITLE
Example with-react-loadable throws errors in the console: Uncaught SyntaxError: Unexpected token < and 

### ISSUE THREAD

**Issue by mbrochh** [jaredpalmer/razzle#403]
When I just `yarn start` the project and open the page in the browser, I'm seeing these errors in the console:

```
Uncaught SyntaxError: Unexpected token <  0.chunk.js:1
Uncaught SyntaxError: Unexpected token <  1.chunk.js:1
Uncaught SyntaxError: Unexpected token <  2.chunk.js:1
Warning: Did not expect server HTML to contain a <img> in <div>. proxyConsole.js:54
```

The page *seems* to work, but surely, there must be a way to get rid of these errors?

**Comment by mbrochh** [comment]
First hint:

The example suggests this code:

```
${chunks
    .map(chunk => `<script src="/${chunk.file}"></script>`)
    .join("\n")}
```

But that would result in a path that cannot be served, because that file should be served via `http://localhost:3001/static/...`

However, when I add that path to the script tag, I'm getting other errors (`webpackJsonp is not defined`)

**Comment by mbrochh** [comment]
I think I accidentally solved this somehow. I followed the README of react-loadable and replaced this:

```
// server.js
const bundles = getBundles(stats, modules);
const chunks = bundles.filter(bundle => bundle.file.endsWith('.js'));

...

${chunks.map(chunk => `<script src="/${chunk.file}"></script>`).join('\n')}
```

with this:

```
// server.js
const bundles = getBundles(stats, modules);

...

${bundles.map(bundle => { return `<script src="${bundle.file}"></script>`; }).join("\n")}
<script>window.main();</script>
```

furthermore, in client.js, I added this:

```
window.main = () => {
  Loadable.preloadReady().then(() => {
```

but now I am having other problems, because it seems that the main() function in the html script tag is called before it is attached to the window in the bundle, which is strange, but I guess I'll figure that one out as well, sooner or later.

**Comment by mbrochh** [comment]
Aha, it seems I have to remove `defer` from the script tags in order to get react-loadable working without errors.

**Comment by begoat** [comment]
@mbrochh  Thank you very much, I also came across the problem that the main function is called too early last night and at that time I didn't know that I should remove `defer` tag to solve the problem. Instead I tried the `with-react-loadable example` given by the owner of this repository.  And I came across `webpackJsonp is not defined` problem too ;) Thanks to you, I solved it just remove the `defer` from script tags.
